php/Blocks/Loader.php

```diff
@@ -126,7 +126,7 @@ public function editor_assets() {
 		wp_enqueue_script(
 			$js_handle,
 			$this->assets['url']['entry'],
-			$js_config['dependencies'],
+			[],
```

Comment:
`npm run dev` and `npm run build` have the `--webpack-no-externals` flag. 

So dependencies are bundled with the JS file and shouldn't be needed here. At least in my testing.